### DISCUSSION TITLE
Change 'show apps' icon

### DISCUSSION THREAD

**Discussion by jewstan83** [SylEleuth/gruvbox-plus-icon-pack#55] [Q&A]
I've looked through the theme folders and can not find where that icon is. How can I change the show apps icon from the penguin to Arch logo?
![Screenshot from 2024-05-02 14-50-02](https://github.com/SylEleuth/gruvbox-plus-icon-pack/assets/131235154/06cb954b-ec82-45df-8984-d7e901c1640b)
The left icon is what I'm seeing for 'show apps' - I really don't like this icon and would rather use the Arch logo.

**Comment by SylEleuth** [comment]
Is it Gnome's `dash-to-dock`? I've seen that icon [here](https://github.com/SylEleuth/gruvbox-plus-icon-pack/issues/21) but I don't know where it comes from. It's not in my pack I think. Despite of similar shape and right background colour I couldn't find it. I tried to reproduce it in my Gnome vm, but dock behaves differently and I don't have time to configure it, but it doesn't display that icon. Maybe ask author [here](https://github.com/SylEleuth/gruvbox-plus-icon-pack/issues/21)?

**Reply by jewstan83** [reply]
Thank you for your reply. It was not a part of your icon pack, it was part of the Gruvbox gnome shell theme. I appreciate your work and your time to reply. If you have a Ko-fi link, let me know and I'll be happy to donate.

**Reply by SylEleuth** [reply]
No worries. I'll leave it open in case someone will come with an answer in the future.
I don't have Ko-fi, nor any other donation page, but thank You for Your generous offer.

**Comment by SylEleuth** [comment]
Could You tell me what are those missing icons? Terminal in the middle and one on the right.

**Comment by jewstan83** [comment]
The terminal is specifically Black Box ([here](https://gitlab.gnome.org/raggesilver/blackbox) and the other is Teams for Linux [here](https://github.com/IsmaelMartinez/teams-for-linux).

**Reply by SylEleuth** [reply]
I've just added icons to the repo. If You are using released versions it should be out in the next month.
